### PR TITLE
Add more information to the github deployments payload

### PR DESCRIPTION
**Pull request by hkjorgensen**
Extend the Github plugin, so it provides more information; "user_email" (?string), "buddy" (?string), "buddy_email" (?string) and "production" (bool)

/cc @zendesk/samson

### Tasks
 - [ ] :+1: from team @zendesk/samson

### CODE REVIEW

File: plugins/github/app/models/github_deployment.rb [hunk 1]
```diff
@@ -26,12 +26,22 @@ def deployment_options
...
     }
   end
 
+  def payload
+    {
+      deployer: @deploy.user.name,
+      deployer_email: @deploy.user.email,
+      buddy: @deploy.buddy_name,
+      buddy_email: @deploy.buddy_email,
+      production: @stage.production?
```
Comment (grosser) [inline plugins/github/app/models/github_deployment.rb]: maybe better to send `environment` ?

Comment (hkjorgensen) [inline plugins/github/app/models/github_deployment.rb]: @grosser I could. I'm just interested in `production?` in our use case. How would the environment look like?

**Comment by grosser** [comment]
hmmm I guess that's not as easy since this might be going to multiple
environments ... if we just want the first it would be
stage.deploy_groups.first.environment.name

**Comment by hkjorgensen** [comment]
Ok thanks, sticking with production for now.

**Comment by grosser** [comment]
are you reading that back via the api or is it visible in github UI ?

**Comment by grosser** [comment]
ok if I change this to ?

```
buddy: {
  name: foo
  email: bar
}
```

**Pull request merged**